alot/ui.py

```diff
@@ -481,15 +481,18 @@ def clear_notify(self, messages):
             self._notificationbar = None
         self.update()
 
-    def choice(self, message, choices=None, select=None, cancel=None,
-               msg_position='above'):
+    def choice(self, message, choices=None, choices_to_return=None,
+               select=None, cancel=None, msg_position='above'):
```

Comment:
I'd really prefer to see the new arguments added to the end of the function signature, there is no guarantee that callers aren't using the keyword arguments as positional, and adding the new arguments to the end ensures we don't accidentally break anything (and our test coverage is less than complete).